.prettierrc

```
@@ -0,0 +1,13 @@
+{
+  "overrides": [
+    {
+      "files": "**/*.scss",
```

We should also format `.css` files. There are 2 in the project at the moment.

```suggestion
      "files": ["**/*.css", "**/*.scss"],
```